File: web_fund/algos/algo15.js
// 15. Print Low, Return High - Create a function that takes in an array of
// numbers. The function should print the lowest value in the array, and
// return the highest value in the array

function printLowReturnHigh(arr) {
    var lowVal = arr[0];
    var highVal = arr[0];
    for (var i = 0; i < arr.length; i++) {
        if (arr[i] < lowVal) {
            lowVal = arr[i];
        }
        if (arr[i] > highVal) {
            highVal = arr[i];
        }
    }
    console.log(lowVal);
    return highVal;
}

console.log(printLowReturnHigh([4, 11, -2, 19, 3, 15]))

File: practice_algos/python/algo15.py
# 15. Print Low, Return High - Create a function that takes in an array of numbers.
# The function should print the lowest value in the array, and return the highest
# value in the array

def low_high(arr):
    low = arr[0]
    high = arr[0]
    for num in arr:
        if num < low:
            low = num
        if num > high:
            high = num
    print(low)
    return high

print(low_high([1,3,5,-7]))